spec/requests/vendor_api/make_an_offer_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - POST /api/v1/applications/:id/offer', type: :request do
+  include VendorApiSpecHelpers
+
+  context 'successfully made a conditional offer' do
```

Comment:
the other contexts are using the format `when a ...`

Reply:
👍 Updated in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/260/commits/d918d189dab55f870753e00e94a0b728da0678f5